pkgs/development/tools/analysis/hopper/default.nix

```diff
@@ -1,6 +1,21 @@
-{ stdenv, fetchurl, pkgs, lib }:
+{ mkDerivation, stdenv, fetchurl, pkgs, lib, qtbase }:
+let
 
-stdenv.mkDerivation rec {
+  ldLibraryPath = with pkgs; stdenv.lib.makeLibraryPath  [
+    libbsd.out
+    libffi.out
+    gmpxx.out
+    python27Full.out
```

Comment:
avoid using python27Full, unless you really need something that is not in python27

Reply:
Also, do not mix `python27Packages` and `python27Full` or you will depend on two interpreter versions.

Reply:
done